#Left image and cam info
sensor_msgs/Image left_image
sensor_msgs/CameraInfo left_info

#Right image and cam info
sensor_msgs/Image right_image
sensor_msgs/CameraInfo right_info